app/src/main/res/drawable/foreground_disabled.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: `foreground_disabled.xml` is a pretty generic name. Can we name it like `preference_multiselect...` da-da-da if it's not used anywhere else?